lib/fluent/plugin/out_detect_exceptions.rb

```diff
@@ -30,8 +30,10 @@ class DetectExceptionsOutput < Output
     config_param :multiline_flush_interval, :time, default: nil
     desc 'Programming languages for which to detect exceptions. Default: all.'
     config_param :languages, :array, value_type: :string, default: []
-    desc 'Maximum number of lines in a detected stack trace. Default: 1000.'
+    desc 'Max. number of lines in combined log entries. Default: 1000.'
```

Comment:
[Optional] This could also be "Maximum number of lines to flush", which would fit on one line... Also below.